lib/bgsite_official_web/templates/layout/root.html.leex

```diff
@@ -31,13 +32,24 @@
       </a>
 
       <form class="form-inline my-2 my-lg-0 btn-toolbar">
- <div class="input-group rounded">
-  <input type="search" class="form-control rounded" placeholder="Search" aria-label="Search"
+ <!--<div class="input-group rounded">
+  <input type="search" class="form-control rounded"  placeholder="Search" aria-label="Search"
     aria-describedby="search-addon" />
   <span class="input-group-text border-0" id="search-addon">
     <i class="fas fa-search"></i>
-  </span>
+  </span>-->
 </div>
+  <class="search">
+  <%= form_for @conn, Routes.websites_path(@conn, :index),[
+    method: :get,
+    class: "pull-right"], fn f -> %>
+    <class="input-group-text border-0" id="search-addon">
+    <%= search_input f, :query%>
+    </class>
+    <%= submit "Search"%>
+  <%end%>
+
+  <% @inner_content %>
```

Comment:
We don't need 2nd inner_content, plz delete it

Reply:
nqma 2ri